packages/ra-ui-materialui/src/input/ResettableTextField.tsx

```diff
@@ -26,8 +26,8 @@ function ResettableTextField(
         value,
         resettable,
         disabled,
-        variant = 'filled',
-        margin = 'dense',
+        variant,
```

Comment:
there are other places where we set the default variant, e.g. in `SelectArrayInput`. Your current PR is incomplete and will break existing apps.